adding project_documentation, preface and fallback content

marcosgopen opened the pull request:
these adoc files are needed to navigate the documentation

jmfinelli commented on src/main/asciidoc/project/fallback_content/conventions.adoc:
````diff
@@ -0,0 +1,162 @@
+
+The above text includes application names; system-wide menu names and items; application-specific menu names; and buttons and text found within a GUI interface, all presented in proportional bold and all distinguishable by context. 
+
+``Mono-spaced Bold Italic`` or [app]```Proportional Bold Italic```
````
```suggestion
`Mono-spaced Bold Italic` or `Proportional Bold Italic`
```

jmfinelli commented on src/main/asciidoc/project/fallback_content/conventions.adoc:
```diff
@@ -0,0 +1,162 @@
+
+____
+Choose menu:System[Preferences > Mouse] from the main menu bar to launch `Mouse Preferences`.
+In the [label]#Buttons# tab, click the [label]#Left-handed mouse# check box and click btn:[Close] to switch the primary mouse button from the left to the right (making the mouse suitable for use in the left hand). 
```
```suggestion
In the `Buttons tab`, click the `Left-handed mouse` check box and click `Close` to switch the primary mouse button from the left to the right (making the mouse suitable for use in the left hand). 
```

jmfinelli replied:
Same for all other occurrences in the text

marcosgopen commented:
thanks Manuel, I'll double check those occurrences

marcosgopen commented:
Occurrences checked and corrected

marcosgopen committed:
adding project_documentation.adoc

adding preface and fallback_content

Update src/main/asciidoc/project/fallback_content/conventions.adoc

Co-authored-by: jmfinelli <61553861+jmfinelli@users.noreply.github.com>
Update src/main/asciidoc/project/fallback_content/conventions.adoc

Co-authored-by: jmfinelli <61553861+jmfinelli@users.noreply.github.com>
quotes convertions

jmfinelli commented on src/main/asciidoc/project/fallback_content/conventions.adoc:
```diff
@@ -0,0 +1,162 @@
+If the remote machine is _example.com_ and your username on that machine is john, type ``ssh john@example.com``. 
+
+The `mount -o remount `file-system`` command remounts the named file system.
+For example, to remount the [path]_/home_ file system, the command is ``mount -o remount /home``. 
```
```suggestion
For example, to remount the `_/home_ file system`, the command is `mount -o remount /home`. 
```

jmfinelli commented on src/main/asciidoc/project/fallback_content/conventions.adoc:
```diff
@@ -0,0 +1,162 @@
+The `mount -o remount `file-system`` command remounts the named file system.
+For example, to remount the [path]_/home_ file system, the command is ``mount -o remount /home``. 
+
+To see the version of a currently installed package, use the `rpm -q `package`` command.
```
```suggestion
To see the version of a currently installed package, use the `rpm -q package` command.
```

marcosgopen committed:
Update src/main/asciidoc/project/fallback_content/conventions.adoc

Co-authored-by: jmfinelli <61553861+jmfinelli@users.noreply.github.com>

marcosgopen committed:
Update src/main/asciidoc/project/fallback_content/conventions.adoc

Co-authored-by: jmfinelli <61553861+jmfinelli@users.noreply.github.com>

jmfinelli commented on src/main/asciidoc/project/fallback_content/conventions.adoc:
```diff
@@ -0,0 +1,162 @@
+
+____
+To connect to a remote machine using ssh, type `ssh `username`@`domain.name`` at a shell prompt.
+If the remote machine is _example.com_ and your username on that machine is john, type ``ssh john@example.com``. 
```
```suggestion
If the remote machine is _example.com_ and your username on that machine is john, type `ssh john@example.com`.
```

marcosgopen committed:
Update src/main/asciidoc/project/fallback_content/conventions.adoc

Co-authored-by: jmfinelli <61553861+jmfinelli@users.noreply.github.com>

jmfinelli commented on src/main/asciidoc/project/fallback_content/conventions.adoc:
```diff
@@ -0,0 +1,162 @@
+For example: 
+
+____
+To connect to a remote machine using ssh, type `ssh `username`@`domain.name`` at a shell prompt.
```
```suggestion
To connect to a remote machine using ssh, type `ssh username@domain.name` at a shell prompt.
```

jmfinelli commented on src/main/asciidoc/project/fallback_content/conventions.adoc:
```diff
@@ -0,0 +1,162 @@
+To connect to a remote machine using ssh, type `ssh `username`@`domain.name`` at a shell prompt.
+If the remote machine is _example.com_ and your username on that machine is john, type `ssh john@example.com`.
+
+The `mount -o remount `file-system`` command remounts the named file system.
```
```suggestion
The `mount -o remount file-system` command remounts the named file system.
```

jmfinelli commented on src/main/asciidoc/project/fallback_content/conventions.adoc:
```diff
@@ -0,0 +1,162 @@
+It will return a result as follows: `package-version-release`. 
+____
+
+Note the words in bold italics above &mdash;username, domain.name, file-system, package, version and release.
```
```suggestion
Note the words in bold italics above -username, domain.name, file-system, package, version and release.
```

jmfinelli commented on src/main/asciidoc/project/fallback_content/feedback.adoc:
```diff
@@ -0,0 +1,11 @@
+
+= We Need Feedback!
+(((feedback,contact information for this manual)))
```
This is not needed I think

marcosgopen replied:
I agree, thanks

jmfinelli commented on src/main/asciidoc/project/fallback_content/feedback.adoc:
```diff
@@ -0,0 +1,11 @@
+
+Please feel free to raise any issues you find with this document in our https://issues.jboss.org/browse/JBTM[issue tracking system]
+
+ifdef::backend-docbook[]
+[index]
+== Index
+// Generated automatically by the DocBook toolchain.
+endif::backend-docbook[]
```
Is this needed?

marcosgopen replied:
probably not, I will remove it.

marcosgopen committed:
Update src/main/asciidoc/project/fallback_content/conventions.adoc

Co-authored-by: jmfinelli <61553861+jmfinelli@users.noreply.github.com>

marcosgopen committed:
Update src/main/asciidoc/project/fallback_content/conventions.adoc

Co-authored-by: jmfinelli <61553861+jmfinelli@users.noreply.github.com>

marcosgopen committed:
Update src/main/asciidoc/project/fallback_content/conventions.adoc

Co-authored-by: jmfinelli <61553861+jmfinelli@users.noreply.github.com>

marcosgopen committed:
feedback update
